test/README

```diff
@@ -6,9 +6,11 @@ files for them.
 To run the tests, you need to install the following packages on
 Fedora:
 
-  # yum install trickle nbd-server npm python-libguestfs qemu mock qemu-kvm python \
+  $ yum install trickle nbd-server npm python-libguestfs qemu mock qemu-kvm python \
```

Comment:
The '#' means execute this as root. If you want to change to '$' then you should probably add sudo to the command.